### DISCUSSION TITLE
How to use a Scala 2.13 library in Scala 3 project?

### DISCUSSION THREAD

**Discussion by informarte** [com-lihaoyi/mill#1341] [Q&A] [answered]
https://scalacenter.github.io/scala-3-migration-guide/docs/compatibility/classpath.html describes how to use a Scala 2.13 library in a Scala 3 project using SBT. Can the same thing achieved with mill?

**Comment by lefou** [comment, marked as answer]
You can use the `withDottyCompat` method:

```scala
ivy"a::b:c".withDottyCompat(scalaVersion())
```
